compass/ocean/tests/spherical_harmonic_transform/QU_convergence/init.py

```diff
@@ -0,0 +1,63 @@
+from compass.model import run_model
+from compass.step import Step
+import subprocess
+
+
+class Init(Step):
+    """
+    A step for an initial condition for for the cosine bell test case
+    """
+    def __init__(self, test_case, resolution, algorithm, order):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.global_convergence.cosine_bell.CosineBell
+            The test case this step belongs to
+
+        resolution : int
+            The resolution of the (uniform) mesh in km
+        """
+
+        super().__init__(test_case=test_case,
+                         name='QU{}_init_{}_{}'.format(resolution,algorithm,order),
```

Comment:
This looks great!  My only comments would be the usual about maybe using f-strings.